CHANGELOG.md

```diff
@@ -21,6 +24,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Deprecated
 
 ### Removed
+- Configuration/PHPUnit/phpunit.xml, using phpunit.xml.dist as default configuration
```

Comment:
I think we don't need this changelog line - the config file has been moved (not removed), and that was part of the new stuff.